content/issues/2023-11-09-maintenance-aks-1.26-publick8s.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Update "public" AKS cluster Kubernetes version to 1.26
+date: 2023-11-09T08:30:00-00:00
+resolved: false
+resolvedWhen: 2023-11-09T10:00:00-00:00
+# Possible severity levels: down, disrupted, notice
+severity: disrupted
+affected:
+  - accounts.jenkins.io
+  - get.jenkins.io
+  - incrementals.jenkins.io
+  - javadoc.jenkins.io
+  - jenkins-wiki-exporter.jenkins.io
+  - ldap.jenkins.io
+  - plugins.jenkins.io
+  - repo.azure.jenkins.io
+  - reports.jenkins.io
+  - stories.jenkins.io
+  - uplink.jenkins.io
+  - www.jenkins.io
+  - ci.jenkins.io
+  - issues.jenkins.io
```

Comment:
A more complete list can be found here (it doesn't include issues.jenkins.io nor ci.jenkins.io though): https://github.com/jenkins-infra/status/blob/3c755b449f064f0206eaf251e8d29b54b4ac6066/content/issues/2023-10-19-publik8s-public-nginx-ingress-arm64-migration.md#L8C1-L31

Reply:
Is issues.jenkins.io on the affected list because the LDAP server may be disrupted by the upgrade?

Reply:
> A more complete list can be found here (it doesn't include issues.jenkins.io nor ci.jenkins.io though): https://github.com/jenkins-infra/status/blob/3c755b449f064f0206eaf251e8d29b54b4ac6066/content/issues/2023-10-19-publik8s-public-nginx-ingress-arm64-migration.md#L8C1-L31

The list has unrelated services (such as `repo.aws.jenkins.io` or `repo.do.jenkins.io` ) but it helped me to add some I missed (PHS for instance).

> Is issues.jenkins.io on the affected list because the LDAP server may be disrupted by the upgrade?

Absolutely, same for ci.jenkins.io. I have mentionned it in the body (along with adding trusted.ci, cert.ci, infra.ci and release.ci for same reasons)